packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -73,6 +75,8 @@ class RunCommand extends RunCommandBase {
         negatable: false,
         hide: true,
         help: 'Stay resident after running the app.');
+    // Hidden option to enable a benchmarking mode.
```

Comment:
Let's be clearer what we mean by "benchmarking". I'm sure developers will discover this option by reading the sources anyway. Are we benchmarking the app? The edit/refresh? Something else?